keps/sig-node/2033-kubelet-in-userns-aka-rootless/README.md

```diff
@@ -654,67 +818,211 @@ Note that this feature gate does not support separating kubelet's user namespace
+logs or events for this purpose.
+-->
 
-* **What are the SLIs (Service Level Indicators) an operator can use to determine 
-the health of the service?**
-  - [ ] Metrics
-    - Metric name:
-    - [Optional] Aggregation method:
-    - Components exposing the metric:
-  - [X] Other (treat as last resort)
-    - Details: Use `systemctl --user is-system-running` to verify whether the processes (RootlessKit, kubelet, kube-proxy, and CRI) are running.
+They can determine if a Pod is running on a node that is running in UserNS.
 
-* **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
+###### How can someone using this feature know that it is working for their instance?
 
-N/A
+<!--
+For instance, if this is a pod-related feature, it should be possible to determine if the feature is functioning properly
+for each individual pod.
+Pick one more of these and delete the rest.
+Please describe all items visible to end users below with sufficient detail so that they can verify correct enablement
+and operation of this feature.
+Recall that end users cannot usually observe component logs or access metrics.
+-->
 
-* **Are there any missing metrics that would be useful to have to improve observability 
-of this feature?**
+- [X] Events
+  - Event Reason:  No CrashLoopBackOff
+- [ ] API .status
+  - Condition name: 
+  - Other field: 
+- [ ] Other (treat as last resort)
+  - Details:
 
-N/A, but it'd be useful to have the kubelet publish whether or not it is running rootless, as a boolean metric.
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+
+<!--
+This is your opportunity to define what "normal" quality of service looks like
+for a feature.
+
+It's impossible to provide comprehensive guidance, but at the very
+high level (needs more precise definitions) those may be things like:
+  - per-day percentage of API calls finishing with 5XX errors <= 1%
+  - 99% percentile over day of absolute value from (job creation time minus expected
+    job creation time) for cron job <= 10%
+  - 99.9% of /health requests per day finish with 200 code
+
+These goals will help you determine what you need to measure (SLIs) in the next
+question.
+-->
+
+99.9% of /health requests per day finish with 200 code
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+<!--
+Pick one more of these and delete the rest.
+-->
+
+- [ ] Metrics
+  - Metric name:
+  - [Optional] Aggregation method:
+  - Components exposing the metric:
+- [X] Other (treat as last resort)
+  - Details: Use `systemctl --user is-system-running` to verify whether the processes (RootlessKit, kubelet, kube-proxy, and CRI) are running.
+
+###### Are there any missing metrics that would be useful to have to improve observability of this feature?
+
+<!--
+Describe the metrics themselves and the reasons why they weren't added (e.g., cost,
+implementation difficulties, etc.).
+-->
+
+No
```

Comment:
See my question about kubelet in rootless vs regular mode.

Reply:
Reworded as:

> 
> [`NodeSystemInfo`](https://pkg.go.dev/k8s.io/api/core/v1#NodeSystemInfo) could have `RunningInUserNS *bool`
> for better observability.
> 
> However, this does not seem necessary, as an admin should have been already aware whether they are going to setup kubelet in UserNS
> when they are adding a new node to the cluster.